Lib/idlelib/autocomplete_w.py

```diff
@@ -256,7 +256,7 @@ def winconfig_event(self, event):
         else:
             # place acw above current line
             new_y -= acw_height
-        acw.wm_geometry("+%d+%d" % (new_x, new_y))
+        self.widget.after(1, acw.wm_geometry, "+%d+%d" % (x, y))
```

Comment:
This should use `new_x` and `new_y`, rather than `x` and `y`.

Reply:
Of course, thank you. This happened in the last change suggestion, I didn't notice.